src/System.Reflection.Extensions/src/project.json

```diff
@@ -13,7 +13,7 @@
         "System.Diagnostics.Contracts": "4.0.0",
         "System.Diagnostics.Debug": "4.0.10",
         "System.IO": "4.0.10",
-        "System.Reflection": "4.1.0-rc3-23901",
```

Comment:
Is there a reason you did this with a package reference instead of a ProjectReference?